### PR TITLE
Adding new RTTA subjects to get an adviser start page

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/aVg2qWZs/2730-returners-update-content-to-cover-widening-of-eligibility-for-rtta-support?filter=member:ellienodder

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
Looks good to me @EllieNodder !

I'm going to mark this as Draft (and not approve it) just to be on the safe side as this can't be merged until we have the go-ahead from the Returners team following their review on 21 Feb.

However, it can definitely be tested with the CRM team in the meantime to make sure the two new subject values are passed through correctly (and the three existing ones).

File: app/views/teacher_training_adviser/steps/_subject_not_found.html.erb
Comment (gemmadallmandfe) [inline app/views/teacher_training_adviser/steps/_subject_not_found.html.erb:7]: Thanks for adding this one in!

Do we need the `(MFL)`? We don't have this anywhere else?

NB I know this is on the original and not something you've added!

**Review by gemmadallmandfe — COMMENTED** [review]
Weirdly I can't see the changes in the Review app - but there isn't a build failure?

File: app/views/teacher_training_adviser/steps/_subject_not_found.html.erb
Comment (EllieNodder) [reply app/views/teacher_training_adviser/steps/_subject_not_found.html.erb:7]: good point, will take it out

**Comment by EllieNodder** [comment]
> Weirdly I can't see the changes in the Review app - but there isn't a build failure?

@gemmadallmandfe is it because it's in draft? (not sure if that makes a difference?)

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by peteryates — APPROVED** [review]
Looks good.

**Pull request merged**
